django/forms/models.py

```diff
@@ -1031,10 +1031,13 @@ def _get_foreign_key(parent_model, model, fk_name=None, can_fail=False):
                 )
             )
         else:
+            fks_name = [fk.name for fk in fks_to_parent]
             raise ValueError(
-                "'%s' has more than one ForeignKey to '%s'." % (
+                "'%s' has more than one ForeignKey to '%s'. "
+                "Choices for ForeignKeys are: %s." % (
```

Comment:
This ticket is about recommending a `fk_name`, so we should rewrite this message, e.g.
```python
raise ValueError(
    "'%s' has more than one ForeignKey to '%s'. You must specify "
    "a 'fk_name'." % (
        model._meta.label,
        parent_model._meta.label,
    )
)
```

Reply:
Ok i will make the changes, and revert the changes made in the tests.